service_container/factories.rst

```diff
@@ -13,9 +13,9 @@ the service container to call a method on the factory rather than directly
 instantiating the class.
 
 Suppose you have a factory that configures and returns a new ``NewsletterManager``
-object::
+object, by calling the static ``createNewsletterManager()`` method::
 
-    class NewsletterManagerFactory
+    class NewsletterManagerStaticFactory
```

Comment:
This class name looks ugly ... but at the same time it's perfectly understandable. I'm divided about it 😕